### PR TITLE
Import system xml 4

### PR DESCRIPTION
**Pull request by atsushieno**

### CODE REVIEW

File: mcs/class/System.XML/Dummy.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Xml.Schema;
+
+namespace System.Xml
```
Comment (marek-safar) [inline mcs/class/System.XML/Dummy.cs]: No need to re-implement this. Include mcs/build/common/SR.cs instead as we do elsewhere

File: mcs/class/System.XML/Dummy.cs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	}
+
+	static class ThisAssembly
+	{
+		public const string Version = "System.Xml";
+	}
+}
+
+// Types within this namespace are taken from referencesource
+namespace System.Xml.Serialization
```
Comment (marek-safar) [inline mcs/class/System.XML/Dummy.cs]: Pleas move this to separate files with meaningful name

File: mcs/class/System.XML/Test/System.Xml.Schema/XmlSchemaTests.cs [hunk 3]
```diff
@@ -537,6 +538,7 @@ public override object GetEntity (Uri absoluteUri, string role, Type ofObjectToR
 		}	
 		
 		[Test]
+		[Ignore (".NET fails as well as referencesource")]
```
Comment (marek-safar) [inline mcs/class/System.XML/Test/System.Xml.Schema/XmlSchemaTests.cs]: Why not to fix such tests to test the actual behaviour

File: mcs/class/System.XML/Dummy.cs (hunk 1 above)
Comment (atsushieno) [inline mcs/class/System.XML/Dummy.cs]: I thought RESOURCE_STRINGS support for multiple files is broken?

File: mcs/class/System.XML/Test/System.Xml.Schema/XmlSchemaTests.cs (hunk 3 above)
Comment (atsushieno) [inline mcs/class/System.XML/Test/System.Xml.Schema/XmlSchemaTests.cs]: The actual behavior is BUG. WE don't have to ASSERT THAT it fails to load.

File: mcs/class/System.XML/Dummy.cs (hunk 1 above)
Comment (marek-safar) [inline mcs/class/System.XML/Dummy.cs]: That's C# code you duplicated

Comment (atsushieno) [inline mcs/class/System.XML/Dummy.cs]: You didn't notice that those in SR.cs are in different namespace. Have you ever tried to actually remove those my classes and build? It won't. There is reason they are there.

Comment (marek-safar) [inline mcs/class/System.XML/Dummy.cs]: It's not different namespace but parent namespace which is fine if the sources use System based namespace which I think they do. I didn't try to build it myself

Comment (atsushieno) [inline mcs/class/System.XML/Dummy.cs]: There is no Res class in SR.cs.

Comment (marek-safar) [inline mcs/class/System.XML/Dummy.cs]: I see, this is intended to be Res class as everywhere else

**Comment by atsushieno** [comment]
build

**Comment by atsushieno** [comment]
build

**Comment by atsushieno** [comment]
build

**Pull request merged**